public_html/wp-content/mu-plugins/blocks/assets/src/blocks-store.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Implements wc-blocks-store.
+ */
```

Comment:
`[MIGHT]` - IIRC, the Core blocks don't have top-level comments like this. If there's an existing convention, we should be consistent with it.

`[SHOULD]` - I'm not necessarily opposed to keeping this, but right now it doesn't feel very useful, since it isn't saying anything that can't be inferred from the file name. If we keep it, it'd probably be better to give a high-level overview of what the store does, and why it's needed.

Reply:
removed top-level common, because its pretty obvious what this do